apps/dashboard/services.py

```diff
@@ -540,9 +623,21 @@ def _format_participant_data(self, participant) -> dict[str, Any]:
             "last_activity": participant.last_activity.isoformat() if participant.last_activity else None,
         }
 
+    def _cache_key(self, filters: dict) -> str:
+        def normalize(obj):
+            if isinstance(obj, dict):
+                return {k: normalize(obj[k]) for k in sorted(obj)}
+            if isinstance(obj, list):
+                return sorted(normalize(v) for v in obj)
+            return obj
+
+        normalized = normalize(filters or {})
+        json_str = json.dumps(normalized, separators=(",", ":"), sort_keys=True, cls=DjangoJSONEncoder)
```

Comment:
`separators=(",", ":")` is the default?

Reply:
It's for compact encoding:
```
json.dumps([1, 2, 3, {'4': 5, '6': 7}], separators=(',', ':'))
'[1,2,3,{"4":5,"6":7}]'
```

Reply:
Oh I missed the space characters. I see the default ones have a space whereas your don't